src/test/regress/expected/isolation_drop_vs_all.out

```diff
@@ -241,7 +241,7 @@ step s1-drop: DROP TABLE drop_hash;
 step s2-distribute-table: SELECT create_distributed_table('drop_hash', 'id'); <waiting ...>
 step s1-commit: COMMIT;
 step s2-distribute-table: <... completed>
-ERROR:  could not create distributed table: relation does not exist
+ERROR:  could not open relation with OID XXXX
```

Comment:
Why does this change? We call `EnsureRelationExists` before the new code?

Reply:
I thin this error comes from a different line and we move the check earlier now, though I am not 100% sure why `EnsureRelationExists` doesn't error out before the below line:

```c
	EnsureCitusTableCanBeCreated(relationId);

	/* enable create_distributed_table on an empty node */
	InsertCoordinatorIfClusterEmpty();

	/*
	 * Lock target relation with an exclusive lock - there's no way to make
	 * sense of this table until we've committed, and we don't want multiple
	 * backends manipulating this relation.
	 */
	Relation relation = try_relation_open(relationId, ExclusiveLock);
	if (relation == NULL)
	{
		ereport(ERROR, (errmsg("could not create distributed table: "
							   "relation does not exist")));
	}

```